bindings/wasm/README.md

```diff
@@ -3,7 +3,7 @@
 > This is the beta version of the official WASM bindings for [IOTA Identity](https://github.com/iotaledger/identity.rs).
 
 ## [API Reference](https://wiki.iota.org/identity.rs/libraries/wasm/api_reference)
-## [Examples](./examples/README.md)
+## [Examples](https://github.com/iotaledger/identity.rs/blob/dev/bindings/wasm/examples/README.md)
```

Comment:
This is a relative link for the GitHub repository but I get that it breaks the wiki.

Maybe we should link to the Wasm examples on the `main` branch instead of `dev` which the root README.md does? 
https://github.com/iotaledger/identity.rs/blob/dev/README.md#getting-started

```suggestion
## [Examples](https://github.com/iotaledger/identity.rs/blob/main/bindings/wasm/examples/README.md)
```